.circleci/config.yml

```diff
@@ -478,9 +478,9 @@ workflows:
       - just_simulate_ink_sep_holocene_system_config_upgrade:
           context:
             - circleci-repo-readonly-authenticated-github-token
-
+      # Simulation sequence for Superchain Version Sync (at the 27th of February 2025). The next task would be 31 but we need to wait until the task is 31 is merged.
+      # tasks: "030 031"
       - simulate_sequence:
           network: "eth"
-          tasks: "021 022 base-003 ink-001"
-          block_number: "21573136" # If not specified, the latest block number is used.
-
+          tasks: "030"
+          block_number: "21936400" # If not specified, the latest block number is used.
```

Comment:
we should definitely use `latest` here since this is for a production task

Reply:
Yes, for this batch of tasks we should definitely have such a process in place! After tasks are executed and we update their status, we'd include this change in the PR